hr/forms.py

```diff
@@ -42,3 +42,22 @@ def __init__(self, *args, **kwargs):
                     choices=WorkDayChoices,
                     initial=WorkDayEnum.WORKING_DAY.name,
                 )
+
+    def clean_employee(self):
+        clean = super().clean()
```

Comment:
```suggestion
        cleaned_data = super().clean()
```
this is default variable name in forms. Your variable names should show exactly what they contain.
clean - is verb, means action. And super() returns data after cleaning, means cleaned_data

Reply:
Done